packages-ui/roosterjs-react/lib/ribbon/plugin/createRibbonPlugin.ts

```diff
@@ -49,6 +50,21 @@ class RibbonPluginImpl implements RibbonPlugin {
      * @param event PluginEvent object
      */
     onPluginEvent(event: PluginEvent) {
+        if (
+            event.eventType == PluginEventType.KeyDown &&
+            event.rawEvent.key == 'k' &&
+            (event.rawEvent.ctrlKey || event.rawEvent.metaKey) &&
+            !event.rawEvent.altKey
+        ) {
+            if (insertLink.disableInsertLink) {
```

Comment:
By saying "adding a check to the feature", I mean you can add code to RibbonPlugin class, for example:
```ts
interface RibbonPluginOptions {
  allowInsertHotKey? boolean;
}

class RibbonPlugin {
  constructor(options?: RibbonPluginOptions) { ... }
}
```

Then you can pass an options parameter to the plugin to enable this feature.

Reply:
I see what you meant there, thank you for clarifying!

Reply:
Updated.